# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/tests/gtest/moz.build
# Commit: ef46ec074d6a
# Full Hash: ef46ec074d6aba10d6e4342e305f36ead6996ddb
# Author: Honza Bambas <honzab.moz@firemni.cz>
# Date: 2018-06-07 21:50:48
# Regressor Bug: 1413868
# File Overlap Count: 1
# Description:
#   Bug 1413868. r=valentin
# ==============================================================================

diff -r 14f7dab3a9bd -r ef46ec074d6a xpcom/tests/gtest/moz.build
--- a/xpcom/tests/gtest/moz.build	Wed Jun 06 14:50:17 2018 -0700
+++ b/xpcom/tests/gtest/moz.build	Thu Jun 07 13:56:16 2018 -0400
@@ -23,6 +23,7 @@
     'TestEventTargetQI.cpp',
     'TestExpirationTracker.cpp',
     'TestFile.cpp',
+    'TestFilePreferencesWin.cpp',
     'TestGCPostBarriers.cpp',
     'TestID.cpp',
     'TestInputStreamLengthHelper.cpp',
